src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,50 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.assertj.core.internal.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void checkSetSize() {
+        Assertions.assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @Test
+    void contains() {
+        Assertions.assertThat(numbers.contains(1)).isTrue();
+        Assertions.assertThat(numbers.contains(2)).isTrue();
+        Assertions.assertThat(numbers.contains(3)).isTrue();
```

Comment:
```suggestion
        Assertions.assertThat(numbers).contains(1);
        Assertions.assertThat(numbers).contains(2);
        Assertions.assertThat(numbers).contains(3);
```

이런 방식도 가능해요 :)

`assertThat(numbers).containsAnyOf(3, 2, 1)` 으로 하면 한 줄로도 확인이 가능해요.